components/omega/src/ocn/OceanState.cpp

```diff
@@ -36,10 +36,18 @@ int OceanState::init() {
    HorzMesh *DefHorzMesh = HorzMesh::getDefault();
    Halo *DefHalo         = Halo::getDefault();
 
-   // This hard-wired variable needs to be updated
-   // with retrivals/config options
    int NVertLevels = 60;
 
+   // Retrieve NVertLevels from Config if available
+   Config *OmegaConfig = Config::getOmegaConfig();
+   Config DimConfig("Dimension");
+   if (OmegaConfig->existsGroup("Dimension")) {
+      Err = OmegaConfig->get(DimConfig);
+      if (DimConfig.existsVar("NVertLevels")) {
+         Err = DimConfig.get("NVertLevels", NVertLevels);
+      }
+   }
```

Comment:
```suggestion
   int NVertLevels = DefHorzMesh->NVertLevels;
```

Reply:
Thanks, actually had this one committed but neglected to push it